src/FeeCollector.sol

```diff
@@ -132,11 +132,6 @@ contract FeeCollector is Ownable {
         SafeTransferLib.safeTransfer(ERC20(_token), msg.sender, withdrawAmt);
     }
 
-    /**
-     * @notice Executes when native is sent to this contract through a non-existent function.
-     */
-    fallback() external payable { } // solhint-disable-line no-empty-blocks
```

Comment:
My current understanding is:
- The `receive()` function allows for ETH transfers to the contract (i.e., transactions with empty input data).
- The `fallback)` function only gets called when ETH is transferred via a transaction that contains input data, but the input data does not match the function signature of any of the functions defined on the contract (i.e., transferring ETH via a non-existent function on the contract).

If we decide that the contract shouldn't receive any ETH, then we can remove `receive()`, `fallback()`, and all `payable` keywords. This would throw an exception if ETH is sent to the contract.

However, I don't see a problem in being able to send ETH to the contract, and having `payable` functions is technically a bit more gas-efficient than non-payable functions.

[Source](https://docs.soliditylang.org/en/latest/contracts.html#special-functions)

Reply:
I didn't know that the `payable` modifier made a function more gas-efficient, do you know why that is? In that case, `payable` should remain if only for the gas saving purposes.

I agree there is no problem in allowing the contract to receive ETH. In theory nobody should be sending ETH to the contract anyway, the only time this would be done to user error of some kind right?